run.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
 const { run } = require("./index.js");
-const { reactSnap } = require(`${process.cwd()}/package.json`);
+const { reactSnap, homepage } = require(`${process.cwd()}/package.json`);
 
-run(reactSnap);
+run(reactSnap, homepage);
```

Comment:
it will be simpler

```js
run({
  publicPath: homepage && url.parse(homepage).pathname,
  ...reactSnap
})
```

and remove additional param from run

Reply:
Ok, it's not really clear how you want to structure things. I'll move this logic into run.js.
Is part of this change too restrictive? Do we want to allow homepage but also a custom public path? Is the "public path from homepage" thing specific to CRA?

Reply:
Nevermind!